lib/nodes/socket.cpp

```diff
@@ -97,6 +100,11 @@ char *villas::node::socket_print(NodeCompat *n) {
   case SocketLayer::UNIX:
     layer = "unix";
     break;
+
+  case SocketLayer::TCP_SERVER:
+  case SocketLayer::TCP_CLIENT:
+    layer = "tcp";
```

Comment:
Can we use `tcp-client` and `tcp-server` here as we do during parsing the options?